crowbot/frontend/static/frontend/script.js

```diff
@@ -114,37 +139,24 @@ $( document).ready(function(){
         }
     });
 
-
-
-
-    questionList = $("#question-queue");
-    function addQuestion(question) {
-        questionList.append(
-            $(question)
-        );
-    }
-
-    function makeListItem(question, datetime) {
-        return "<li>" + "Question: " + question  + " Time: " + datetime + "</li>"
-    }
-
     var q_list_root = '/api/question_queue';
 
-    function addQuestionsToList(course_code){
+    function addPendingQuestions(course_code, listmanager){
         $.ajax({
         //Get the question
             url: q_list_root + "/" + course_code,
             method: "GET"
         }).then(function(questions){
             for (q of questions){
-                addQuestion(
-                    makeListItem(q.text, q.datetime)
-                );
+                var content = listmanager.prettyDatetime(q.datetime) + " " + q.text;
```

Comment:
I prefer the layout used in the chat window, i.e. timestamp below the actual text, and smaller; it gives more room for the actual content. Should we use the same here?